website/screens/theme-generator/themes/schemas/opinionated.schema.json

```diff
@@ -64,6 +64,9 @@
   "link": {
     "baseColor": "color"
   },
+  "navTabs": {
+    "baseColor": "color"
+  },
```

Comment:
I would remove this until the opinionated theme is done.

Reply:
removed!